docs/platforms/aws-lambda/nodejs/index.md

```diff
@@ -0,0 +1,465 @@
+
+Once initialized, each instance of your function can process thousands of requests without performing another Initialization. However, AWS Lambda function instance containers will shutdown within 5-15 minutes of inactivity. After that, the next event will be a Cold-Start, causing Initialization to run again.
+
+### Tags
+
+These are the Tags attached to this Span:
+
+```javascript
+/* Tags: Standard */
+
+s.span.id: "ja819kasfj10asfh" // This Span's ID
+s.span.parent.id: "e8a4e2a264145041" // This Span's parent Span ID.
+s.span.timestamp: "2022-07-13T20:10:48.794945024Z" // This Span's time.  Same as start_time.
+s.span.start_time: "2022-07-13T20:10:48.794945024Z" // This Span's start time.
+s.span.end_time: "2022-07-13T20:10:48.799375104Z" // This Span's end time.
+
+/* Tags: aws.lambda */
+ 
+aws.lambda.initialization.duration: 600 // Maps to the Cloudwatch Logs Report "Init Duration"
+```
+
+## `aws.lambda.invocation`
+
+This is a Span within a Trace for AWS Lambda. After Initialization, Extensions and the handler of the AWS Lambda function run in the Invocation phase. This phase includes:
+
+Running External Extensions in parallel with the function. These also continue running after the function has completed, enabling Serverless Console to capture diagnostic information and ingest metrics, traces and logs.
+
+Running the wrapper logic for Internal Extensions.
+
+Running the handler for your AWS Lambda.
+
+The Invocation phase is comprised mostly of your handler (i.e. your business logic), and you want to optimize that as best you can because its performance (combined with Initialization performance) will have the biggest impact on the experience for your users and customers.
+
+Serverless Console provides a lot of auto-instrumentation for measuring Spans within your business logic, which is located within the Invocation span, such as requests to other AWS Services and HTTP calls generally.
+
+It's important to note that your function's timeout setting limits the duration of the entire Invocation phase. For example, if you set the function timeout as 360 seconds, the function and all extensions need to complete within 360 seconds.
+
+### Tags
+
+These are the Tags attached to this Span:
+
+```javascript
+/* Tags: Standard */
+
+s.span.id: "91ashafj0a91hasf" // This Span's ID
+s.span.parent.id: "e8a4e2a264145041" // This Span's parent Span ID.
+s.span.timestamp: "2022-07-13T20:10:48.794945024Z" // This Span's time.  Same as start_time.
+s.span.start_time: "2022-07-13T20:10:48.794945024Z" // This Span's start time.
+s.span.end_time: "2022-07-13T20:10:48.799375104Z" // This Span's end time.
+
+/* Tags: aws.lambda */
+
+aws.lambda.invocation.duration: 432 // Maps to the Cloudwatch Logs Report "Duration"
```

Comment:
It'll be good clarify if `aws.lambda` wraps just single invocation, or all that happens in given lambda process.

If it's the latter, then I'd expect it more as:

```
aws.lambda.processes[].invocation.duration: 432
```